webportal/src/app/market_detail/components/summary.jsx

```diff
@@ -111,13 +106,6 @@ export default function Summary(props) {
         iconProps: { iconName: 'Copy' },
         onClick: showCopyPopup,
       },
-      {
-        key: 'editItem',
```

Comment:
>editItem [](http://example.com/codeflow?start=14&length=8)

 💡 [Discussion]

I am just testing if the latest CodeFlow version is compatible with GitHub, so if you see this, please simply ignore it.

Reply:
👍 👎 
🔴 🟡 🟢 
❔ 💡 🐞 
🚀 👀 😎